mlflow/exceptions.py

```diff
@@ -1,5 +1,32 @@
+import json
+
+from mlflow.protos.databricks_pb2 import INTERNAL_ERROR, ErrorCode
+
+
 class MlflowException(Exception):
     """Base exception in MLflow."""
+    def __init__(self, message, **kwargs):
```

Comment:
Can we replace `**kwargs` with an `error_code` keyword arg with default value `INTERNAL_ERROR`? Currently, the `error_code` variable name always needs to be specified with the parameter.

Reply:
Yeah, will do.